windows-nsis/openvpn.nsi

```diff
@@ -30,6 +30,10 @@ SetCompressor lzma
 !addplugindir .
 !include "nsProcess.nsh"
 
+; DotNetChecker.nsh to detect whether .net 4.0 is enabled, which is required for openvpnserv2 ( https://github.com/ReVolly/NsisDotNetChecker )
+!addplugindir .
```

Comment:
Based on [NSIS documentation](http://nsis.sourceforge.net/Docs/Chapter5.html#addplugindir) !addplugindir makes NSIS scan a directory (here ".") for plugins. After the scan NSIS should see all of the plugins (here .nsh files) in the current directory. Unless you know something I don't, I'd say scrap the second !addplugindir command, and just keep the one above _!include "nsProcess.nsh"_.